src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/Connection.php

```diff
@@ -119,7 +120,7 @@ public static function fromDsn(string $dsn, array $options = [], HttpClientInter
         }
 
         $parsedPath = explode('/', ltrim($parsedUrl['path'] ?? '/', '/'));
-        if (\count($parsedPath) > 0) {
```

Comment:
previous code were buggy. explode always return an not empty array.
we should assert here that queue name is != ampty string